tests/test_integration.py

```diff
@@ -30,6 +30,8 @@ def create_test_database():
         database_url = DatabaseURL(url)
         if database_url.dialect == "mysql":
             url = str(database_url.replace(driver="pymysql"))
+        elif database_url.driver == "aiopg":
```

Comment:
Let's check the `.scheme` here instead in these conditions.

Reply:
fixed